i18n/MonotoneMuseum/ko_kr.yml

```
@@ -25,65 +25,538 @@ ko_kr:
+          【세계사】
+          허무가 나타나 세계가 붕괴한다. 아무것도 없는 공간에, 등장해있던 캐릭터들이 내던져진다. 난이도 12의 【재봉】판정에 성공한 캐릭터는 파멸에서 벗어날 수 있다. 실패한 캐릭터는 행방불명이 된다. 엔딩 페이즈로.
+
+      WDT2:
+        name: 세계왜곡표ver2.0
+        items:
+        - |-
+          【일그러짐의 덤불】
+          씬 전체가 강철보다도 단단한 가시 덤불로 메워진다. 비행상태가 아닌 캐릭터는 이동할 때마다 <찌르기> 3D6+[그 씬에 등장한 캐릭터 중 최대 배우 레벨]점의 대미지를 입는다.
+        - |-
+          【세계왜곡】
+          세계왜곡표（『MM』P264）를 굴린다.
+        - |-
+          【뒤집힌 세계】
+          하늘이 땅으로, 땅이 하늘로 바뀐다. 그 씬 동안 모든 비행상태인 캐릭터는 비행 상태가 아닌 것으로 취급하고, 비행상태가 아닌 캐릭터는 비행상태로 취급한다.
+        - |-
+          【바다의 기억】
+          존재할 리 없는 파도소리가 들리고 세계 전체가 파도에 삼켜진다. 이 씬 중 비행상태가 아닌 모든 캐릭터는 수중상태로 취급한다.
+        - |-
+          【공간만곡】
+          세계가 굽는다. 이곳과 저곳이 의미를 잃는다. 이 씬 동안 모든 캐릭터의 이동력이 ∞이 되며 특기와 장비의 사정거리는 모두 「시야」가 된다.
+        - |-
+          【농무세계】
+          우유보다도 하얗고 농후한 안개가 세계를 닫는다. 이 씬 동안 특기와 장비의 사정거리가 「시야」「6m이상」인 경우 모두 「5m」로 변경된다.
+        - |-
+          【폭우】
+          무시무시한 비가 내린다. 비는 모든 사람의 마음을 얼어붙게 만든다. 이 씬 동안 「구분：총」인 무기를 사용한 경우 펌블치가 +6이 된다. 또한 다른 인게이지에 대한 물리공격의 달성치가 -2된다.
+        - |-
+          【질척질척】
+          어느틈엔가 발 밑이 부정형의 진흙탕이 된다. 이 씬 동안 비행상태가 아닌 캐릭터가 같은 인게이지에 물리공격을 할 경우 펌블치가+6이 된다.
+        - |-
+          【폭풍】
+          바람이 몰아치며 운명을 비웃는다. 이 씬 동안 비행상태가 아닌 캐릭터가 같은 인게이지에 물리공격을 할 경우 펌블치가+6이 된다.
+        - |-
+          【마법폭주】
+          마법의 힘이 폭주해 더는 누구도 감당할 수 없다. 세계는 끝난다. 사라진다. 이 씬 동안 「구분 : 술」의 특기를 사용한 판정의 펌블치가 +6이 된다.
+        - |-
+          【자기사】
+          거대한 올풀림이 발생해 치명적인 대미지를 입힌다. 가장 박탈치가 높은 PC 한 명이 전투불능이 된다. 다수의 PC가 해당하는 경우 GM이 랜덤으로 결정한다.
+        - |-
+          【허무의 응보】
+          실잣는이에 의해 계속해서 일그러져왔던 인과가 역습한다. 이 씬에서 《허구출현》을 사용한 경우 대가에 더해 10D6의 실대미지를 입는다. 이 대미지는 경감할 수 없으며 대신 받을 수도 없다.
+        - |-
+          【허구의 끝】
+          신이 버린 것인가 신을 버린 것인가. 이탈의 끝에 세계가 끝장난다. 당신은 이 씬에서 이탈능력을 사용한 경우 상승한 박탈치의 10배만큼 실 대미지를 입는다. 이 대미지는 경감할 수 없으며 대신 받을 수도 없다.
+        - |-
+          【인격교대】
+          혼, 엄밀히 말하면 혼의 배후에 있는 무언가가 뒤바뀐다. 왼쪽의 플레이어에게 캐릭터 시트를 넘기고 이 신 동안 그 플레이어가 당신의 PC를 연기한다. GM이 인정한다면 GM도 NPC를 바꾼다.
+        - |-
+          【인과의 실】
+          비웃는 소리가 들린다. 「이탈의 결과는 공유하자고？」 PC 전원의 박탈치의 평균을 내 (소수점 올림) 전원의 박탈치를 그 계산결과로 바꿔쓴다.
+        - |-
+          【검에 죽다】
+          사람을 벌한 응보를, 검에는 검의 대가를. 이 씬에서 공격을 하고 펌블했을 경우 5D6의 실데미지를 입는다.
+        - |-
+          【비밀폭로】
+          등장해있는 모든 실잣는이, 이형, 가람은 숨기고싶은 비밀(그 캐릭터의 주관으로 OK) 하나를 밝힌다. 그러지 못할 경우 10D6의 실대미지를 받는다. 이 대미지는 경감할 수도 대신 받을 수도 없다.
+        - |-
+          【황폐의 왕】
+          깨달았을 때는 모든 캐릭터가 허무의 황야의 황폐의 왕의 몸 위에 있다. 모든 캐릭터는 그대로 허무에 삼켜져 5D6의 실데미지를 입는다. 이 데미지는 경감할 수도 대신 받을 수도 없다. 황폐의 왕이 당신들에게 어떤 영향을 미칠지는 GM이 정한다.
+      OT2:
+        name: 징조표ver2.0
+        items:
+        - |-
+          【신념 상실】
+          당신의 정신이 당신의 과거를 부정한다. 당신은 [출신]과 그에 해당하는 기억을 그 씬 동안 상실한다. 특성으로 인한 효과는 잃지 않지만 그것을 인식하는 것은 불가능하다.
+        - |-
+          【졸도】
+          더 이상 당신은 이탈의 부담을 버티는 것이 불가능하다. 그것이 세계에 거스른 대가이다. 당신은 [전투불능]이 된다. 이미 [전투불능]일 경우 [사망]한다.
+        - |-
+          【육체 붕괴】
+          유리가 깨지는 소리가 나며, 당신도 깨져간다. 당신은 2D6+캐릭터레벨 점의 【HP】를 잃는다. 
```

The trailing space should be removed.
```suggestion
          유리가 깨지는 소리가 나며, 당신도 깨져간다. 당신은 2D6+캐릭터레벨 점의 【HP】를 잃는다.
```